examples/qml_with_threaded_logic/src/lib.rs

```diff
@@ -124,6 +115,17 @@ mod website {
         }
     }
 
+    impl RustObj {
+        fn process_event(&mut self, event: &Event, cpp: &mut CppObj) {
```

Comment:
This needs to be in the `impl Qt<MyObject>` block, not in the `impl RustObj` block, as we need access to the C++ functionality.

Reply:
Actually this one is a rust private method (as Event isn't a supported type), what happens is that the handle_update_request calls this method and passes in CppObj. But i've added support for both anyway.

Reply:
Yes, but it will in future still need to be implemented in the `impl MyObjectQt` block (e.g. `impl cxx_qt::QObject<T>`).

That doesn't mean it needs to be put in the CXX bridge, so it will still be rust-private.

Reply:
In my opinion, this still needs to be in `impl cxx_qt::QObject<RustObj>`, as it modifies a C++ property (e.g. `cpp.set_title`).